ekip/nationalparks/data/pass-list-20150828.csv

```diff
@@ -177,7 +177,7 @@ Modoc NF - Warner Mountain District,530-279-6116,Cedarville,CA,http://www.fs.fed
 Mojave National Preserve,760-252-6108,Kelso,CA,http://www.nps.gov/moja/index.htm,YES,YES,YES
 "Muir Woods National Monument (see also Golden Gate NRA, CA)",415-388-7368,Mill Valley,CA,http://www.nps.gov/muwo,YES,YES,YES
 Needles BLM Field Office,760-326-7000,Needles,CA,http://www.blm.gov/ca/st/en/fo/needles.html,YES,YES,YES
-New Melones Lake - Reclamation Central California Area Office,209-536-9094,Sonora,CA,http://www.usbr.gov/mp/ccao/newmelones/index.html ��,YES,YES,YES
+New Melones Lake - Reclamation Central California Area Office,209-536-9094,Sonora,CA,http://www.usbr.gov/mp/ccao/newmelones/index.html Â ,YES,YES,YES

```

Comment:
There are some encoding related issues here. Can we ensure that they don't affect anything?